microsoft/testsuites/core/floppy.py

```diff
@@ -30,6 +37,14 @@ class Floppy(TestSuite):
         priority=1,
     )
     def check_floppy_module(self, node: RemoteNode) -> None:
+        if (
+            isinstance(node.os, CentOs)
+            and node.os.information.version.major < 7
+            and node.os.information.version.major < 7
```

Comment:
Should it be minor? And it can be compared like `version < "7.7.0"`. BTW, it means the 7.7 is out of life, it should be `version <= "7.7.0"`.